bin/cleanup_old_instances.rb

```diff
@@ -1,12 +1,15 @@
 target_date = 3.months.ago
-old_instances = Kuroko2::JobInstance
+count = 0
+
+Kuroko2::JobInstance
   .where('finished_at < ?', target_date)
   .or(Kuroko2::JobInstance.where('canceled_at < ?', target_date))
-
-count = old_instances.count
-
-Kuroko2::JobInstance.transaction do
-  old_instances.destroy_all
+  .order(id: :asc)
+  .in_batches do |old_instances|
+  count += old_instances.count
```

Comment:
.size ?

Reply:
in_batches returns `ActiveRecord::Relation`, so I think `count` is better, isn't it?

Reply:
ah yes correct; the following lines just run `DELETE` statement. Please disregard

Reply:
💭 I forgot how Rails `destroy_all` behaves... if it tries to load records, `.size` might be better

Reply:
If we can reduce a count query by calling `size` after/before calling `destroy_all`, using `size` looks great.  Otherwise, we should use `count` as normal. In addition, we should leave a comment about that behavior.

Reply:
in_batches yields loaded active record relation 🙈
so, not to send count query, fix to use `.size` f59f230

Reply:
cc @taiki45 https://github.com/cookpad/kuroko2/pull/108#discussion_r177965061

Reply:
🙈